extra-network/kubo/autobuild/build

```diff
@@ -0,0 +1,9 @@
+abinfo "Building Kubo..."
+make build
+cmd/ipfs/ipfs commands completion bash > "$SRCDIR"/ipfs-completion.bash
```

Comment:
Absolute path here. `"$SRCDIR"/cmd/ipfs/ipfs ...`, it is also recommended to wrap wide lines.